packages/core/config/webpack.common.js

```diff
@@ -169,11 +174,6 @@ const getSharedPlugins = (isEnvProduction) => {
       ),
       ...getEnvVars(),
     }),
-    new Dotenv({
-      path: path.resolve(redwoodPaths.base, '.env'),
-      silent: true,
-      ignoreStub: true, // FIXME: this might not be necessary once the storybook webpack 4/5 stuff is ironed out. See also: https://github.com/mrsteele/dotenv-webpack#processenv-stubbing--replacing
-    }),
```

Comment:
Per discussion:
- ignoreStub: true is no longer needed for Storybook; I've already removed in `main`
- Let's add this back and not change config in this PR
- but do keep tests; we need 'em!